### ISSUE TITLE
Capture intervals shorter than 1 minute

### ISSUE THREAD

**Issue by svenha** [alessio/camorama#9]
Is there a way to specify an automatic capture interval like 0.1 minutes?

**Comment by Fluffkin** [comment]
Yes. Sort of. Edit the camorama.glade file. Change the gtkspinbutton "interval_entry" values. Digits:1 Adjustment: 1 0.1 360 0.1 10 0
That gives you steps of 6 seconds

**Comment by svenha** [comment]
Thanks, that works perfectly!

**Comment by githubwalt** [comment]
I have the same question, but can't find a .glade file!  So I guess that this file is no longer used since this is an old thread.  How can we do this now please?

**Comment by svenha** [comment]
There is a file `data/camorama-gtk4.ui` (and `data/camorama-gtk3.ui` ) with `interval_entry`. Does changing its `climb-rate` value work?

**Comment by githubwalt** [comment]
@svenha 
I think I will have to build it then with the modified file?  Haven't done that yet.
Meanwhile I'm using Cheese (which is somewhat the default for Ubuntu) with burst mode and have set that to 3 seconds, 20 pictures.  So that way I get 20 pictures in 1 minute.
